edb/common/compiler.py

```diff
@@ -76,17 +99,17 @@ def _current(self):
     current = property(_current)
 
 
-class Counter:
+class SimpleCounter:
     def __init__(self):
         self.counts = collections.defaultdict(int)
 
-    def nextval(self, name='default'):
+    def nextval(self, name: str = 'default') -> int:
         self.counts[name] += 1
         return self.counts[name]
 
 
-class AliasGenerator(Counter):
-    def get(self, hint=None):
+class AliasGenerator(SimpleCounter):
+    def get(self, hint: Optional[str]=None) -> str:
```

Comment:
If an empty string can be a sentinel, it's generally preferred. In this case could 'v' be the default?

Reply:
Fixed.